### PR TITLE
Test to verifiy PVC can be expanded more than once

### PR DESCRIPTION
**Pull request by jilju**
OCS-302 - FT-OCP-PVCExpand- Validate PVC expand of already expanded PVC

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        )
+
+        self.pods = pods_cephfs + pods_rbd
+
+        # Set volume mode on PVC objects
+        for pvc_obj in self.pvcs_cephfs+self.pvcs_rbd:
+            pvc_info = pvc_obj.get()
+            setattr(pvc_obj, 'volume_mode', pvc_info['spec']['volumeMode'])
+
+    def expand_and_verify(self, pvc_size_new):
```
Comment (ebenahar) [inline tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:62]: can this method be used also in tests.manage.pv_services.test_pvc_resize.TestPvcResize.test_pvc_resize?
I think the validation here is relevant also for OCS-325

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.5')
+@pytest.mark.polarion_id('OCS-302')
+class TestPvcExpand(ManageTest):
```
Comment (ebenahar) [inline tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:16]: can we consolidate the methods in this class with tests.manage.pv_services.test_pvc_resize.TestPvcResize?

Comment (jilju) [reply tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:16]: The plan is to add more tier1 test cases cases under this test class. I will try to add larger test cases in separate PRs. I will also move test_pvc_resize to this module in a separate PR.

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py (hunk 1 above)
Comment (jilju) [reply tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:62]: I will move test_pvc_resize to this module in a separate PR.

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py (hunk 2 above)
Comment (ebenahar) [reply tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:16]: +1 thanks!

**Pull request merged**